client/keys/import_test.go

```diff
@@ -94,7 +94,7 @@ HbP+c6JmeJy9JXe2rbbF1QtCX1gLqGcDQPBXiCtFvP7/8wTZtVOPj8vREzhZ9ElO
 			t.Cleanup(cleanupKeys(t, kb, "keyname1"))
 
 			keyfile := filepath.Join(kbHome, "key.asc")
-			require.NoError(t, os.WriteFile(keyfile, []byte(armoredKey), 0644))
+			require.NoError(t, os.WriteFile(keyfile, []byte(armoredKey), 0o644))
```

Comment:
typo?

Reply:
I don't think so.  I think that they're the same thing and `gofumpt` simplified it.

Not 100% sure.

Reply:
wierd tho.  Googling.

Reply:
https://github.com/golang/go/issues/33363

Reply:
oh I didnt know this. Learned something new today, thank you